scripts/build_renewable_profiles.py

```diff
@@ -420,6 +424,27 @@ def get_hydro_capacity_annual_hydro_generation(
         if "natura" in config and config["natura"]:
             excluder.add_raster(paths.natura, nodata=0, allow_no_overlap=True)
 
+            natura = rio.open(paths.natura)
```

Comment:
The more I think to it, the more I think that at user-level, the check should be done between the africa_shape and the cutout and natura separately.
An error shall be thrown if the africa_shape is not contained in both cutout and natura.

If the two shapes do not overlap it's not an issue

Reply:
@davide-f Agree. From the user's perspective we should care about the area of interest. Apart of that, if the provided natura.tiff does not match the africa_shape, we should recomment to use the default global natura.tiff

Reply:
If you don't have any objections, I'd open a new PR with a check of a proper match between the region and natura.tiff & cutout

Reply:
I'm ok with that; the comparison between natura and cutout is more for developers and it may actually harm the user:
if the two do not overlap, but the countries are contained in both, it's ok.